Dokan/Dokan/Scenes/ProductDetails/Views/InfoSellerView/InfoSellerView.xib

```diff
@@ -0,0 +1,121 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
A lot of constraints with fixed constant values which will not fit for different screen sizes. We need to consider enhancing the view before merging it.